cmd/server/main.go

```diff
@@ -15,6 +15,7 @@ func main() {
 }
 
 func run() error {
-	fmt.Println("http://localhost:" + port)
-	return server.ListenAndServe(port)
+	addr := ":" + port
+	fmt.Println("http://localhost" + addr)
```

Comment:
I do not mind the change, but I'm just curious, why did you add `:` to `addr` instead of keeping `port` and having `:` in the hard-coded string in `fmt.Println()`?

Reply:
It fixes a mistake made in a previous commit: `server.ListenAndServe(port)` is not valid and crashes because it requires a leading `:`.
Storing it in `addr` only allows to avoid making the same computation twice (`Println` + `server.ListenAndServe`)